docs/user-guide/paragraph-section.md

```diff
@@ -0,0 +1,46 @@
+# Paragraph Section
+**********************
+
+The Paragraph Section allows to insert a textual content to the story. The story editor can also click on the <img src="../img/button/++++.jpg" class="ms-docbutton"/> button to add additional contents to this section (like media, other parafraphs or embed third party contens). It is possible to chose between:
```

Comment:
The Paragraph Section allows to insert a textual content to the story. The story editor can also click on the <img src="../img/button/++++.jpg" class="ms-docbutton"/> button to add additional contents to this section (like media, other paragraphs or embed third party contens). It is possible to chose between: